File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinTecsMerchantServices SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinTecsMerchantServicesMakeContext
{
    public static function call(array $ctxmap, ?BluefinTecsMerchantServicesContext $basectx): BluefinTecsMerchantServicesContext
    {
        return new BluefinTecsMerchantServicesContext($ctxmap, $basectx);
    }
}

File: py/bluefintecsmerchantservices_sdk/utility/make_context.py
# BluefinTecsMerchantServices SDK utility: make_context

from bluefintecsmerchantservices_sdk.core.context import BluefinTecsMerchantServicesContext


def make_context_util(ctxmap, basectx):
    return BluefinTecsMerchantServicesContext(ctxmap, basectx)
